src/audioPlayer.js

```diff
@@ -100,14 +109,49 @@ async function setPlayer() {
 }
 
 function resetControllers() {
-    AUDIO_TOGGLES.forEach((audioToggle) => {
-        const playIconPlayer = audioToggle.querySelector(
-            '[data-element=play]'
-        )
-        const pauseIconPlayer = audioToggle.querySelector(
-            '[data-element=pause]'
-        )
-        playIconPlayer.setAttribute('display', 'block')
-        pauseIconPlayer.setAttribute('display', 'none')
+    [...AUDIO_TOGGLES, ...VIDEO_TOGGLES].forEach((mediaToggle) => {
+        const playIcon = mediaToggle.querySelector('[data-element=play]')
+        const pauseIcon = mediaToggle.querySelector('[data-element=pause]')
+        const unmuteIcon = mediaToggle.querySelector('[data-element=unmute]')
+        const muteIcon = mediaToggle.querySelector('[data-element=mute]')
+
+        if (playIcon) playIcon.setAttribute('display', 'block')
+        if (pauseIcon) pauseIcon.setAttribute('display', 'none')
+        if (unmuteIcon) unmuteIcon.setAttribute('display', 'block')
+        if (muteIcon) muteIcon.setAttribute('display', 'none')
+    })
+}
+
+
+/**
+ * A helper function to toggle a video's mute state. Bind the trigger to this function so that the video selector can be
```

Comment:
Since this function only takes care of the mute/unmute and not the play pause, does the video auto plays in a loop on the page?

Reply:
Correct, I can add a comment to clarify :)